llmfoundry/models/layers/ffn.py

```diff
@@ -27,10 +27,13 @@
 except ModuleNotFoundError:
     is_te_imported = False
 
-try:
-    import megablocks
-    is_megablocks_imported = True
-except ModuleNotFoundError:
+if torch.cuda.is_available():
```

Comment:
This should solve the issue on `compose-rl`, but another solution would be to simply to
```
try:
    import megablocks
    is_megablocks_imported = True
except:
    is_megablocks_imported = False
```
to catch any straggling issues?

Reply:
I like it as is, it's more intentional and specific. Can you add a note explaining that we are gating this because megablocks depends on triton and triton will error upon import if you do not have gpus